stories/examples/playground.stories.tsx

```diff
@@ -209,6 +210,7 @@ export const Plugins = () => {
     withLink(),
     withList(options),
     withDeserializeHTML({ plugins }),
+    withMarks({ shouldChange: false }),
```

Comment:
Instead of using the option globally (since we want to trigger `onChange` for other operations), can you use the option in `autoformatInline`?